react/features/whiteboard/actionTypes.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Configure the whiteboard collaboration details.
+ * {{
+ *      type: SETUP_WHITEBOARD,
+ *      collabDetails
+ * }}
+ */
+export const SETUP_WHITEBOARD: string = 'SETUP_WHITEBOARD';
+
+/**
+ * Open the whiteboard collaboration session.
+ * {{
+ *      type: OPEN_WHITEBOARD,
+ *      participantId
+ * }}
+ */
+export const OPEN_WHITEBOARD: string = 'OPEN_WHITEBOARD';
+
+/**
+ * Close the whiteboard collaboration session.
+ * {{
+ *      type: CLOSE_WHITEBOARD
+ * }}
+ */
+ export const CLOSE_WHITEBOARD: string = 'CLOSE_WHITEBOARD';
```

Comment:
I thought we had agreed that the whiteboard will never be really closed. Just hidden, so we can call that closed, but I'm not sure why we need both actions.

Reply:
I see. Maybe to avoid mistakes we can call that "reset" ?